tokio/src/sync/watch.rs

```diff
@@ -918,7 +918,7 @@ impl<T> Sender<T> {
             return Err(error::SendError(value));
         }
 
-        self.send_replace(value);
+        self.send_modify(|old| *old = value);
```

Comment:
This changes the behavior on panic in the destructor of `old`.

Reply:
Then let's better leave it as is. Without personal use cases for `send()` I don't have any stakes here.